lib/floe/container_runner/docker.rb

```diff
@@ -182,8 +182,8 @@ def docker_event_status_to_event(status)
 
       def inspect_container(container_id)
         JSON.parse(docker!("inspect", container_id).output).first
-      rescue
-        nil
+      rescue AwesomeSpawn::CommandResultError => err
```

Comment:
Not for this PR, but it's weird to me that docker! method fails and we catch AwesomeSpawn::CommandResultError, as that requires the caller to know the implementation of the callee.  Does it make more sense to have `docker!` do the rescue/catch and raise something?

Reply:
That being said, it's all self contained in the same file, so probably not a big deal.